src/NeuralNetwork/NeuralNetworkUtils.js

```diff
@@ -11,6 +11,10 @@ class NeuralNetworkUtils {
    */
   // eslint-disable-next-line class-methods-use-this
   normalizeValue(value, min, max) {
+    // When min is equal to max, set everything to 0
+    if (min === max) {
+      return 0;
```

Comment:
So just thinking about this a bit, should we log some sort of warning in the console? Perhaps we should also just return `min` (or `max`) to keep the original values? And log that they cannot be normalized?

Reply:
I think a warning would be helpful! I am not sure how to word it though. Perhaps "The input parameter <x> cannot be normalized because all values for that parameter are identical. The NeuralNetwork can still be trained, but please check your input data".

If we keep the original (non-zero) data, that parameter effectively becomes a bias value for the next layer since it is constant. If we zero that parameter we are pretty much deleting it from the neural network. I'm not sure whether keeping it could throw off the training, but I'm leaning toward just zeroing the parameter since it doesn't contribute to training anyway.

Reply:
Ah, I see what you mean! How about:

```js
console.warn("Normalization failed: All values for this feature are identical (min === max). The data for this feature will be set to 0, effectively removing it from the model. Please check your input data.");
```

What do you think?

Reply:
Sounds good! I will work on this.

Reply:
Great! let me know when it is ready to merge! Maybe we can do a release after this one?

Reply:
Ready to merge!